src/hipscat/io/write_metadata.py

```diff
@@ -174,8 +180,9 @@ def write_parquet_metadata(catalog_path):
     )
     subschema = subschema.remove(subschema.get_field_index(paths.DIR_DIRECTORY_PREFIX))
 
-    metadata_path = os.path.join(catalog_path, PARQUET_METADATA_FILENAME)
-    common_metadata_path = os.path.join(catalog_path, PARQUET_COMMON_METADATA_FILENAME)
+    catalog_base_dir = file_io.get_file_pointer_from_path(catalog_path)
+    metadata_path = paths.get_parquet_metadata_pointer(catalog_base_dir)
+    common_metadata_path = paths.get_common_metadata_pointer(catalog_base_dir)
 
     pq.write_metadata(subschema, metadata_path, metadata_collector=metadata_collector)
```

Comment:
It feels inconsistent that we're wrapping the calls to write a csv from pandas, but not parquet calls to write new parquet files.

Reply:
Ah you're right, I'll add the wrappers for these